share/completions/gradle.fish

```diff
@@ -45,7 +45,7 @@ function __cache_or_get_gradle_completion
     end
     mkdir -m 700 -p $XDG_CACHE_HOME/gradle-completions
 
-    set -l hashed_pwd (fish_md5 -s $PWD)
+    set -l hashed_pwd (__fish_md5 -s $PWD)
```

Comment:
The file was renamed but the function was not, so there isn't actually a __fish_md5 function.